ext/CreateParametersExt.jl

```diff
@@ -335,21 +339,12 @@ function EnergyHydrologyParameters(
     K_sat::F,
     S_s::F,
     θ_r::F,
-    PAR_albedo_dry::SF = nothing,
-    NIR_albedo_dry::SF = nothing,
-    PAR_albedo_wet::SF = nothing,
-    NIR_albedo_wet::SF = nothing,
-    PAR_albedo::SFD = 0.2,
-    NIR_albedo::SFD = 0.4,
-    albedo_calc_top_thickness::TD = 0.07,
+    albedo = Soil.ConstantTwoBandSoilAlbedo(
+        CP.float_type(toml_dict)(0.2),
+        CP.float_type(toml_dict)(0.4),
+    ),
```

Comment:
We could also have 0.2 and 0.4 as default values in the `ConstantTwoBandSoilAlbedo` struct that can be optionally overwritten. This might tie into cleaner design/setup for the user.

Reply:
ok! I added this.